go/tools/release-notes/release_notes.go

```diff
@@ -488,36 +489,39 @@ func groupAndStringifyPullRequest(pr []prInfo) (string, error) {
 }
 
 func main() {
-	from := flag.String("from", "", "from sha/tag/branch")
-	to := flag.String("to", "HEAD", "to sha/tag/branch")
-	versionName := flag.String("version", "", "name of the version (has to be the following format: v11.0.0)")
-	summaryFile := flag.String("summary", "", "readme file on which there is a summary of the release")
-	flag.Parse()
+	var (
+		from, to, versionName, summaryFile string
+	)
+	pflag.StringVar(&from, "from", "", "from sha/tag/branch")
+	pflag.StringVar(&to, "to", "HEAD", "to sha/tag/branch")
+	pflag.StringVar(&versionName, "version", "", "name of the version (has to be the following format: v11.0.0)")
+	pflag.StringVar(&summaryFile, "summary", "", "readme file on which there is a summary of the release")
```

Comment:
That's a good point. We don't necessarily need this since the only caller of this script is the Makefile. I added the shortflags in 569112fc5f anyway just in case :)